manifests/install/pe_gem.pp

```diff
@@ -1,11 +1,13 @@
 # This class links the r10k binary for PE
-class r10k::install::pe_gem {
+class r10k::install::pe_gem (
+  $puppet_master,
+) {
 
   require git
 
   # Puppet Enterprise 3.8 ships code to manage this symlink on install
   # This conditional should not effect FOSS customers based on the fact
-  unless versioncmp($::pe_version, '3.8.0') >= 0 {
+  unless versioncmp($::pe_version, '3.8.0') >= 0 and $puppet_master {
```

Comment:
I was assuming that since r10k wasn't installed by puppet_enterprise that the symlink wouldn't be there as well.  I can test that out to validate.